src/Symfony/Component/Notifier/Bridge/SmsBiuras/README.md

```diff
@@ -0,0 +1,26 @@
+SmsBiuras Notifier
+==================
+
+Provides [SmsBiuras](https://www.smsbiuras.lt) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+SMSBIURAS_DSN=smsbiuras://UID:TOKEN@default?from=FROM&test=0
+```
+
+where:
+- `UID` is your client code
```

Comment:
How does they name this value? UID? ClientCode?

Reply:
In my case it is 2454

Reply:
No I mean "How does the documentation of SmsBiuras name this thing?"

Reply:
Ok UID is fine as they use it in their documentation 👍🏻